c/crossmemory.c

```diff
@@ -4050,6 +4050,31 @@ static bool isModulePrivate(void) {
   return false;
 }
 
+#ifdef _LP64
+#pragma linkage(BPX4GTR,OS)
+#define BPXGTR BPX4GTR
+#else
+#pragma linkage(BPX1GTR,OS)
+#define BPXGTR BPX1GTR
+#endif
+
+static int checkOMVS(int *bpxRC, int *bpxRSN) {
+
+  int returnValue = 0;
+
+  const int timerRealType = 0;  /* ITIMER_REAL */
+  char result[32] = {0};        /* BPXYITIM */
+  void *resultAddr = result;
+
+  /* Use a getitimer() call to check OMVS. */
+  BPXGTR(&timerRealType, &resultAddr, &returnValue, bpxRC, bpxRSN);
```

Comment:
There is a callable service that is designed with a query option to test if the user has access to OMVS [BPXnQDB - query dub](https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.3.0/com.ibm.zos.v2r3.bpxb100/qdb.htm)